doc/api-docs/conf.py

```diff
@@ -235,45 +230,33 @@ class MockQGroupBox:
 EXCLUDED_MODULES = [
     '_core',  # anything ending in '_core'
     'recmodel_server',
-    'recmodel_workspace_fetcher'
+    'recmodel_workspace_fetcher',
+    'natcap.invest.ui'
 ]
 MODEL_ENTRYPOINTS_FILE = os.path.join(DOCS_SOURCE_DIR, 'models.rst')
-
 # Find all importable modules with an execute function
 # write out to a file models.rst in the source directory
 all_modules = {}
-for _loader, name, _is_pkg in itertools.chain(
-        pkgutil.walk_packages(path=[INVEST_LIB_DIR]),  # catch packages
-        pkgutil.iter_modules(path=[INVEST_LIB_DIR])):  # catch modules
-
-    if (any([name.endswith(x) for x in EXCLUDED_MODULES]) or
-        name.startswith('natcap.invest.ui')):
+for _, name, _ in pkgutil.walk_packages(path=[INVEST_LIB_DIR],
+                                        prefix='natcap.'):
+    if any(x in name for x in EXCLUDED_MODULES):
         continue
 
-    try:
-        module = importlib.import_module(name)
-    except Exception as ex:
-        print(ex)
-        continue
```

Comment:
Removed this because I think we want to raise an error if a module doesn't import.

Reply:
I agree, better to fail loudly.